protocol/avdl/chat1/common.avdl

```diff
@@ -509,6 +511,11 @@
 
   record RpInherit {}
 
+  record RpEphemeral {
+    // Messages must be exploding at have at most this lifetime.
```

Comment:
at->and

Reply:
will change in https://github.com/keybase/keybase/pull/2716